test/test_mps.py

```diff
@@ -1866,6 +1866,10 @@ def helper(shape, value):
         helper((2, 8, 4, 5), 0.1)
         helper((2, 3, 4, 5), 0.2)
         helper((2, 8, 4, 5), 0.2)
+        # Mixed types
```

Comment:
This is surprising that these types are not tested through Op Db testing. can you please add testing for Integer types as well.